pkg/check/zones_test.go

```diff
@@ -111,15 +174,16 @@ func teardownTagAttachmentTest(checkContext CheckContext) error {
 	return nil
 }
 
-func setupTagAttachmentTest(checkContext CheckContext, finder *find.Finder, attachmentMask int64) error {
-	tagMgr := checkContext.TagManager
+func setupTagAttachmentTest(checkContext CheckContext, vCenter *VCenter, finder *find.Finder, attachmentMask int64) error {
+	tagMgr := vCenter.TagManager
 	ctx := checkContext.Context
 	if attachmentMask&tagTestCreateRegionCategory != 0 {
 		categoryID, err := tagMgr.CreateCategory(ctx, &vapitags.Category{
 			Name:        "openshift-region",
 			Description: "region tag category",
 		})
 		if err != nil {
+			fmt.Println("Error 1")
```

Comment:
Are these debug prints necessary?

Reply:
Good catch.  There are no longer needed.  It was used for testing to make sure I was able to hit the errors.  I'll clean these up.